ZM compile error with gcc 6.1

```
[  1%] Building CXX object src/CMakeFiles/zm.dir/zm_rtp_ctrl.cpp.o
In file included from /root/ZM/ZM2/src/zm_rtp_ctrl.cpp:24:0:
/root/ZM/ZM2/src/zm_rtp_ctrl.h:128:23: error: flexible array member 'RtpCtrlThread::RtcpPacket::<anonymous union>::Bye::srcN' in an otherwise empty 'struct RtpCtrlThread::RtcpPacket::<anonymous union>::Bye'
         uint32_t srcN[];   // list of sources
                       ^
/root/ZM/ZM2/src/zm_rtp_ctrl.h:126:14: note: in the definition of 'struct RtpCtrlThread::RtcpPacket::<anonymous union>::Bye'
       struct Bye
              ^~~
src/CMakeFiles/zm.dir/build.make:710: recipe for target 'src/CMakeFiles/zm.dir/zm_rtp_ctrl.cpp.o' failed
make[2]: *** [src/CMakeFiles/zm.dir/zm_rtp_ctrl.cpp.o] Error 1
CMakeFiles/Makefile2:291: recipe for target 'src/CMakeFiles/zm.dir/all' failed
make[1]: *** [src/CMakeFiles/zm.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```

This fixes the problem for me:

```diff
diff --git a/src/zm_rtp_ctrl.h b/src/zm_rtp_ctrl.h
index 2526f22..439f444 100644
--- a/src/zm_rtp_ctrl.h
+++ b/src/zm_rtp_ctrl.h
@@ -123,7 +123,7 @@ private:
       } sdes;

       // BYE
-      struct Bye
+      struct
       {
         uint32_t srcN[];   // list of sources
         // can't express trailing text for reason (what does this mean? it's not even english!)

--- a/src/zm_rtp_ctrl.h
+++ b/src/zm_rtp_ctrl.h
@@ -123,7 +123,7 @@ private:
       } sdes;
 
       // BYE
-      struct Bye
+      struct
       {
         uint32_t srcN[];   // list of sources
         // can't express trailing text for reason (what does this mean? it's not even english!)

